docs/docs/integrations/libraries/dbt/reference.md

```diff
@@ -580,27 +581,63 @@ Metadata fetching methods such as <PyObject section="libraries" object="core.dbt
 
 ### Upstream dependencies
 
-#### Defining an asset as an upstream dependency of a dbt model
+#### Defining an asset as an upstream data dependency of a dbt model
 
-Dagster allows you to define existing assets as upstream dependencies of dbt models. For example, say you have the following asset with asset key `upstream`:
+Dagster allows you to define existing assets as upstream data dependencies of dbt models. For example, say you have the following asset with asset key `upstream`:
 
 <CodeExample
   startAfter="start_upstream_dagster_asset"
   endBefore="end_upstream_dagster_asset"
   path="docs_snippets/docs_snippets/integrations/dbt/dbt.py"
 />
 
-Then, in the downstream model, you can select from this source data. This defines a dependency relationship between your upstream asset and dbt model:
+You can define that asset as a source in your `sources.yml` file:
+
+```yaml
+sources:
+  - name: dagster
+    tables:
+      - name: upstream
+```
+
+Then, in the downstream model, you can select from this source data. This defines a data dependency relationship between your upstream asset and dbt model:
 
 ```sql
 select *
   from {{ source("dagster", "upstream") }}
  where foo=1
 ```
 
+#### Defining an asset as an upstream temporal dependency of a dbt model
```

Comment:
Is "upstream temporal" redundant? I feel like I haven't seen that language used elsewhere in the docs instead of just "upstream"

Reply:
It's a bit awkward, but I was trying to distinguish it from a data dependency, as described in this ticket: https://linear.app/dagster-labs/issue/DOC-994/update-dbt-docs-for-defining-asset-as-upstream-dependency-of-a-dbt

* **Data dependency:** An upstream Dagster asset creates data for the dbt model to read
* **Temporal dependency:** An upstream Dagster asset needs to be materialized before a dbt model is scheduled, but it is not the case that the Dagster asset creates data for the dbt model to be read, i.e., the dependency is simply one of sequencing.

Reply:
Forgot to mention, the reason I'm distinguishing between the two is that you have to use slightly different syntax in the dbt model depending on the scenario.